CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ option (USE_BOOST_WAVE "Use Boost Wave for C preprocessor (alternative is to use
 option (USE_QT "Include Qt support" ON)
 option (USE_OPTIX "Include OptiX support" OFF)
 set (OPTIX_EXTRA_LIBS CACHE STRING "Extra lib targets needed for OptiX")
+set (CUDA_TARGET_ARCH "sm_35" CACHE STRING "CUDA GPU architecture (e.g. sm_35)")
```

Comment:
Just out of curiosity: what is it that requires sm_35?

Reply:
The funnel shift intrinsic used to implement left rotate was added in SM35. It seems to me a like a decent middle ground to use as a default, and the CMake variable should hopefully allow users to choose another architecture if needed.